src/main/java/org/folio/search/utils/SearchResponseUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.search.utils;
+
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.folio.search.domain.dto.FolioCreateIndexResponse;
+import org.folio.search.domain.dto.FolioIndexOperationResponse;
+import org.folio.search.domain.dto.FolioIndexOperationResponse.StatusEnum;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class SearchResponseUtils {
+
+  /**
+   * Creates positive {@link FolioCreateIndexResponse} object with list of created indices.
+   *
+   * @param indices list of created indices
+   * @return created {@link FolioCreateIndexResponse} object
+   */
+  public static FolioCreateIndexResponse getSuccessFolioCreateIndexResponse(List<String> indices) {
+    var response = new FolioCreateIndexResponse();
+    response.setIndices(indices);
+    response.setStatus(FolioCreateIndexResponse.StatusEnum.SUCCESS);
+    return response;
+  }
+
+  /**
+   * Creates negative {@link FolioCreateIndexResponse} with error status and list of indices.
+   *
+   * @param indices list of indices that was not created
+   * @return created {@link FolioCreateIndexResponse} object
+   */
+  public static FolioCreateIndexResponse getErrorFolioCreateIndexResponse(List<String> indices) {
+    var response = new FolioCreateIndexResponse();
+    response.setIndices(indices);
+    response.setErrorMessage("error");
```

Comment:
Can we populate it with the actual error?

Reply:
yes, I will add it